src/citrine/resources/project.py

```diff
@@ -221,6 +221,34 @@ def share(self,
             "resource": {"type": resource_type, "id": resource_id}
         })
 
+    def transfer_resource(self, receiving_project_uid: Union[str, UUID],
```

Comment:
This is very minor and optional, but I might suggest swapping the order of these two arguments.  "Transfer {resource} to {recieving_project_uid}"

Reply:
I agree! that's more intuitive